EvidenceApi/V1/Domain/EvidenceRequest.cs

```diff
@@ -29,7 +29,7 @@ public class EvidenceRequest : IEntity
         public List<string> DocumentTypes { get; set; }
 
         [Column("service_requested_by")]
```

Comment:
We should also rename the database column

Reply:
👍